### ISSUE TITLE
Stats Improvement: There's no focus highlight/style on the global settings icon

### ISSUE THREAD

**Issue by kangzj** [Automattic/wp-calypso#78218]
https://github.com/Automattic/wp-calypso/assets/390760/484d6bcb-8709-4cd5-952c-50323016afbf
